api/apps/graphql/resolvers.js

```diff
@@ -88,17 +88,30 @@ export const channels = (root, args, req, ast) => {
 
 export const display = (root, args, req, ast) => {
   return new Promise((resolve, reject) => {
-    Curation.mongoFetch({ '_id': ObjectId(DISPLAY_ID) }, (err, results) => {
-      if (err) {
-        reject(new Error(err))
-      }
+    Curation.mongoFetch({
+      '_id': ObjectId(DISPLAY_ID)
+    }, (err, { results }) => {
+      if (err) { reject(new Error(err)) }
+
```

Comment:
There was some discussion yesterday with @damassi on not using single line conditionals for readability -- minor -- up to you